src/Z3/Base.hs

```diff
@@ -1866,6 +1868,7 @@ marshalMkQ z3_mk_Q ctx pats x s body = marshal z3_mk_Q ctx $ \f ->
 -- variable with index 0, the second to last element of /xs/ refers to the
 -- variable with index 1, etc.
 mkForall :: Context
+          -> Int        -- ^ quantifiers are associated with weights indicating the importance of using the quantifier during instantiation. By default, pass the weight 0.
```

Comment:
Prefer to not introduce breaking changes unless it's really necessary. I think we should rename `mkForall` (and so on) as `mkForallW` and then make `mkForall` a simple call to `mkForallW` with the weight argument set to `0`.

Reply:
I thought about that too, lets do it then :+1: